### PR TITLE
[gatsby-source-wordpress] Do not assume entities will exist

### PR DESCRIPTION
**Pull request by sebastienfi**
Introduced by https://github.com/gatsbyjs/gatsby/pull/4538 the ability to exclude routes. 
If one excludes tags and taxonomies from the routes, the plugin should still build.

### CODE REVIEW

**Comment by dangerbot-gatsby** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  
Gatsby requires contributors to sign each commit.

You can do this for your branch with `git rebase --signoff`, or for a single commit with `git commit --amend -s`.

Check out the [Developer Certificate of Origin docs](https://www.gatsbyjs.org/docs/how-to-contribute/#developer-certificate-of-origin) or comment here if you've any questions.

  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/gatsby-source-wordpress/src/normalize.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ exports.mapTypes = entities => {
 exports.mapAuthorsToUsers = entities => {
   const users = entities.filter(e => e.__type === `wordpress__wp_users`)
   return entities.map(e => {
-    if (e.author) {
+    if (users && e.author) {
```
[the remark this answers is not in the document]
Comment (jbolda) [reply packages/gatsby-source-wordpress/src/normalize.js]: @pieh I believe you are correct.

**Comment by m-allanson** [comment]
@sebastienfi I'd love to get this merged if you have a chance to update it?

**Comment by sebastienfi** [comment]
@m-allanson Sorry guys, got AFK a few days.
I'll get over the prioritized work and come back to this asap.

**Comment by m-allanson** [comment]
@sebastienfi No problem :) thanks for your ongoing contributions!

**Pull request merged**